views/css/bootstrap-prestashop-ui-kit.css

```diff
@@ -2,7 +2,7 @@
 /* not used */
 /* TODO change padding */
 @import url("https://fonts.googleapis.com/css?family=Open+Sans:300,400,500,600,700,800");
-@import url(jquery.pstagger.css);
```

Comment:
Can you explain why we commented this ?

Reply:
I have re-instated this as it is causing errors in the console, but not breaking functionality. 
` [8e8aa02](/blockonomics/prestashop-plugin/pull/58/commits/8e8aa02ec6478847ccb5449298f2cc78cf9198eb)`